.github/workflows/build-swift.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Build Swift Package
+
+on:
+  push:
+    branches:
+      - "rc"
+      - "hotfix-rc"
+      - "main"
+
+jobs:
+  version:
+    name: Get Version
+    runs-on: ubuntu-22.04
+    outputs:
+      package_version: ${{ steps.retrieve-version.outputs.package_version }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+
+      - name: Get Package Version
+        id: retrieve-version
+        run: |
+          VERSION=$(grep -o '^version = ".*"' crates/bws/Cargo.toml | grep -Eo "[0-9]+\.[0-9]+\.[0-9]+")
+          echo "package_version=$VERSION" >> $GITHUB_OUTPUT
+
+  build:
+    name: Build
+    runs-on: macos-13
+    needs: version
+    env:
+      _VERSION: ${{ needs.version.outputs.package_version }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+
+      - name: Install rust
+        uses: dtolnay/rust-toolchain@bb45937a053e097f8591208d8e74c90db1873d07 # stable
+        with:
+          toolchain: stable
+
+      - name: Cache cargo registry
+        uses: Swatinem/rust-cache@a95ba195448af2da9b00fb742d14ffaaf3c21f43 # v2.7.0
+        with:
+          key: sdk-swift-cargo
+
+      - name: Rustup target
+        run: |
+          rustup target install aarch64-apple-ios-sim
+          rustup target install aarch64-apple-ios
+          rustup target install x86_64-apple-ios
+
+      - name: Build
+        shell: bash
+        working-directory: languages/swift
+        id: build
+        run: |
+          chmod +x build.sh
+          ./build.sh
+
+          # SHA Short
+          echo "short-sha=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+
+      - name: Upload BitwardenFFI.xcframework artifact
+        uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+        with:
+          name: BitwardenFFI-${{ env._VERSION }}-${{ steps.build.outputs.short-sha }}.xcframework
+          path: languages/swift/BitwardenFFI.xcframework
+          if-no-files-found: error
```

Comment:
This seems to upload every commit from main in our artifactory, but it's never used?

Reply:
Oh, I see what you mean. Yes, currently the trigger here https://github.com/bitwarden/sdk/pull/268/files/e336370aa3e6b19b4167b200c2e0df48285b4c2d#diff-055e2da2434f07c0a866c511bfa44f89960b26b18915681bd5a97ed5a8689b9fR9 is going to create an artifact that will not be used in the release workflow. If there is no value in having a GitHub artifact from `main`, I can remove it.

Reply:
From my perspective the main value is automatically pushing commits from main to the sdk-swift repository. As this repository will get more activity depending on devops to trigger development releases whenever a new feature gets merged will be tedious

Reply:
https://github.com/bitwarden/sdk/pull/268/commits/a15a57aac19eb951861546a9ef0649149b3e0314
Does this now, I removed release process items around it, and made it simply push to the repository when a build happens on `main`. This adheres to the requirement to use GitHub releases to store the artifact.